source/System.cpp

```diff
@@ -704,6 +705,20 @@ void System::LoadObject(const DataNode &node, Set<Planet> &planets, int parent)
 			object.offset = child.Value(1);
 		else if(child.Token(0) == "object")
 			LoadObject(child, planets, index);
+		else if(child.Token(0) == "defense" && child.Size() >= 2)
+		{
+			object.launcher = GameData::Outfits().Get(child.Token(1));
+			if(child.Size() > 2)
+				object.reload = child.Value(2);
+			if(!object.reload)
+				child.PrintTrace("Reloadtime out of expected range:");
+			else
+				object.reload = 1. / object.reload;
+			if(child.Size() > 3)
+				object.maxAmmo = (int)child.Value(3);
+		}
+		else if(child.Token(0) == "rotate" && child.Size() >= 1)
```

Comment:
Coinciding with the above comment, `turn`, and it should be a child of `defense` since it is only pertinent to StellarObjects with defined defense.